src/assets/20-tech-terms/20-beginner-tech-terms.md

```diff
@@ -0,0 +1,157 @@
+
+### 9. Unit Testing:
+`Unit testing` is a testing method in which every component of software is tested to check if it is working accurately in those smaller components. It means catching bugs and other types of errors closer to the time they are introduced which results in having more reliable code and easier maintenance.
+Key Points:
+- **Isolation**: Tests are written for small units of code like functions/methods, separately from other parts of the application.
+- **Automation**: With unit testing, tests are automated and can be executed continuously as often as required to fix issues immediately.
+- **Refactoring Safety**: Verify that changes made don't break the previous functionality of the application.
+
+### 10. Regression Testing:
+Suppose you've patched a bug or added any code/tests to your software. Great work! What if that fix/addition broke something else that previously worked fine?  That's where `Regression testing` shines. It's a safety net that ensures recently made changes haven't caused any unintended regressions, or reintroduced old bugs.
+
+So, nearly like checking your work before saving your changes in a document, `regression testing` is performing old test cases on a new release to ensure that previously implemented functionality still works as it needs to, after changes have been made. This catches any regressions right away and prevents them from reaching users.
+
+## Automation and Deployment:
+
+### 11. CI/CD (Continuous Integration/Continuous Deployment):
+CI/CD sounds pretty complicated, but it will completely rock your world if you can manage to get it working. It automates the testing and deployment of your code using something called a `pipeline`. A `pipeline` is pretty much a conveyor belt for your code; it tests every change you make and makes it available for deployment to production. This automation helps to reduce time and minimize errors that can slow down your development process.
+
+### 12. Docker:
+Docker helps you bundle your code and its dependencies into a package which is like a secret handshake. No matter where you deploy it, this package ensures that your code runs exactly as you designed. These packages are called `Docker images`. From a single image, you can create isolated environments called `Docker containers`. Each container acts like its mini-pc, sharing resources efficiently. This consistency makes the flow of development, testing and deployment so much easier - all a breeze!
+
+### 13. Cron Jobs:
+Cron jobs are simply time-based tools used to automate repetitive tasks. You can also define the frequency and timing of an actionable task by using a specific syntax called the `cron expression`. This improves productivity and makes sure that critical procedures are carried out in a consistent way.
+### Some use-cases for Cron Jobs
```

Comment:
```suggestion
#### Some use-cases for Cron Jobs
```
I think this should be a H4 or alternatively you could just make it bold as you have done for other subtitles within a paragraph...